<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>391</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLogin">
       <property name="text">
        <string>Войти</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnButton1">
       <property name="text">
        <string>Кнопка1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnButton2">
       <property name="text">
        <string>Кнопка2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnButton3">
       <property name="text">
        <string>Кнопка3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnButton4">
       <property name="text">
        <string>Кнопка4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnButton5">
       <property name="text">
        <string>Кнопка5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnButton6">
       <property name="text">
        <string>Кнопка6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNewForm">
       <property name="text">
        <string>Новое окно</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="QtWidgetsApplication1.qrc"/>
 </resources>
 <connections/>
</ui>
